code/api/src/modules/user/mutations.js

```diff
@@ -19,6 +19,36 @@ export const userSignup = {
       type: GraphQLString
     },
 
+    bio: {
+      name: 'bio',
+      type: GraphQLString
+    },
+
+    image: {
+      name: 'image',
+      type: GraphQLString
+    },
+
+    street: {
+      name: 'street',
+      type: GraphQLString
+    },
+
+    city: {
+      name: 'city',
+      type: GraphQLString
+    },
+
+    state: {
+      name: 'state',
+      type: GraphQLString
+    },
+
+    zip: {
+      name: 'zip',
+      type: GraphQLString
+    },
+
     password: {
```

Comment:
Does this mutation suggest that All of this information is required on registration? Our understanding on the FE is that we were not gonna change anything about the registration and just update it from the Edit Profile page?
@mlynch5187 @sienna-kopf @Strompy what say ye?

Reply:
You're so right!! This info should only be in the Update

Reply:
You are absolutely right. We will remove bio, image, street, city, state, zip from the userSignup mutation in our next PR.